modules/packages/Python.chpl

```diff
@@ -555,6 +555,62 @@ module Python {
 
       Py_Finalize();
     }
+
+    /*
+      Import a Python module. This is equivalent to calling ``import modName``
+      in Python.
+
+      See :proc:`Module.init` for more information.
```

Comment:
If `Module.init` is no longer the preferred way to create a Python module to track, I think we should move the main information it provides in its documentation to here and have `Module.init` reference `importModule` instead.

Reply:
There are other spots that make these references, I'll let you decide what to do with them rather than pinging on all of them with the same message